task/04-arrays-tasks.js

```diff
@@ -454,7 +480,14 @@ function getIdentityMatrix(n) {
  *     3, 3   => [ 3 ]
  */
 function getIntervalArray(start, end) {
-   throw new Error('Not implemented');
+   const array = Array(end - start + 1).fill(start);
```

Comment:
Use `new Array` instead of `Array`. It is a potential bug. Why? If you see MDN, you will not find anything about `Array` function.